templates/search-result.tmpl

```diff
@@ -136,12 +136,24 @@
     
     <div class="navigation">
       
-      {if $previous >= 0}
+    {if $previous >= 0}
 	<span class="txtPrev">
-	  <a href="#" onclick="javascript:jumpto({$previous}); return false;">&laquo; Previous</a>
+	  <a href="#" onclick="javascript:jumpto({$previous}); return false;">&laquo;&nbsp;Previous</a>
 	</span>
 	{/if}
-
+    {for $f in range(max(0, floor($previous/$limit) - 4), 
```

Comment:
The hard coded numbers -4 and 5 are related to the number of Pages that is shown to the user and not to the step. I made This decision to show always 5 pages ahead and before the page that is already displayed